### PR TITLE
feat: Provide introduction and background on KEDA in CNCF Graduation proposal

### PR DESCRIPTION
**Pull request by tomkerkhove**
Provide introduction and background on KEDA in CNCF Graduation proposal

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Relates to https://github.com/kedacore/governance/issues/63

### CODE REVIEW

File: proposals/graduation/keda.md [hunk 1]
```diff
@@ -1,6 +1,131 @@
...
+
+And there are major KEDA adopters from the gaming and retail industry that we are eager to talk about, but cannot share them publicly because of legal reasons. However, we should be able to get in contact for anonymous interviews if there is interest.
+
+### Making KEDA more accessible to contributors
+
+KEDA has a growing community of adopters and contributors that help to make KEDA better every day.
+
+To help our community keep on doing that, we've been working hard to make KEDA more accessible to contributors in a few areas:
+
+- We've made it simpler to get started by **expanding our [contribution guide](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md)**, improve our [docs for adding new scalers](https://github.com/kedacore/keda/blob/main/CREATE-NEW-SCALER.md) as well as make it simpler to do documentation improvements by [providing more documentation in our README](https://github.com/kedacore/keda-docs)
```
Comment (JorTurFer) [inline proposals/graduation/keda.md:125]: Maybe we should add here that we have improved our docs about [how to debug KEDA locally](https://github.com/kedacore/keda/blob/main/BUILD.md#debugging-with-vs-code).  
We are the only project I have seen who explains how to debug metrics server (I reviewed a lot of projects looking for the way to do it, and finally I did it debugging the source code).

**Review by JorTurFer — COMMENTED** [review]
You have done an awesome job @tomkerkhove. 
Congrats!!!
I only left one comment inline and another suggestion here (because I'm not sure about the correct place): Maybe we should attach a picture about the community increase (insights page in the repo). Ignoring summer, the contributions have grown during the last year
image

**Review by sfc-gh-jhollan — APPROVED** [review]
Awesome job. The tone of the proposal is strong

**Review by jeffhollan — APPROVED** [review]
Approving from my personal GitHub :)

**Review by ahmelsayed — APPROVED** [review]

File: proposals/graduation/keda.md (hunk 1 above)
Comment (tomkerkhove) [reply proposals/graduation/keda.md:125]: Great suggestion, I will add it!

**Review by JorTurFer — APPROVED** [review]
Awesome job, congrats!! 🙇

File: proposals/graduation/keda.md [hunk 2]
```diff
@@ -1,6 +1,141 @@
...
+
+As a CNCF project, KEDA integrates with the CNCF landscape and is built on top of the following CNCF projects.
+
+This is important to us as we want to build an extensible and open ecosystem that is easy to integrate with, and re-use existing technologies instead of re-inventing them.
+
+Here is a list of the CNCF projects that KEDA integrates with and some that we are planning to add in the next couple of months:
+
+- The scaler catalog supports a [variety of cloud-native technologies](https://keda.sh/docs/latest/scalers/) including Prometheus & NATS from the CNCF or Cassandra & Apache Kafka from the Apache Software Foundation
+- KEDA provides [Prometheus metrics](https://keda.sh/docs/latest/operate/prometheus/) to operate the autoscaling infrastructure
+- [Artifact Hub provides support for adding external KEDA scalers](https://artifacthub.io/docs/topics/repositories/keda-scalers/) which are [discoverable in our documentation](https://keda.sh/docs/latest/scalers/)
```
Comment (zroubalik) [inline proposals/graduation/keda.md:44]:
```suggestion
- [Artifact Hub provides support for adding external KEDA scalers](https://artifacthub.io/docs/topics/repositories/keda-scalers/) which are [discoverable in our documentation](https://keda.sh/docs/latest/scalers/)
- [Knative](https://knative.dev) is evaluating an integration with KEDA to enable [autoscaling of Knative Eventing infrastructure](https://github.com/knative-sandbox/eventing-autoscaler-keda).
```

**Review by zroubalik — APPROVED** [review]
LGTM

File: proposals/graduation/keda.md (hunk 2 above)
Comment (tomkerkhove) [reply proposals/graduation/keda.md:44]: As agreed
```suggestion
- [Artifact Hub provides support for adding external KEDA scalers](https://artifacthub.io/docs/topics/repositories/keda-scalers/) which are [discoverable in our documentation](https://keda.sh/docs/latest/scalers/)
- [Knative](https://knative.dev/) is prototyping an integration with KEDA to enable [autoscaling of Knative Eventing infrastructure](https://github.com/knative-sandbox/eventing-autoscaler-keda).
```

**Comment by tomkerkhove** [comment]
Let's do this 🚀

**Pull request merged**